src/tests/issues.targets

```diff
@@ -3271,6 +3271,87 @@
         </ExcludeList>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(RuntimeFlavor)' == 'mono' and '$(RuntimeVariant)' == 'minijit' and '$(TargetArchitecture)' == 'arm64'">
+        <ExcludeList Include="$(XunitTestBinBase)/Interop/SuppressGCTransition/SuppressGCTransitionTest/**">
+            <Issue>needs triage</Issue>
```

Comment:
For this and below, do we know why these are failing to add GH issues for each? If not, to get them tracked, add all failing tests to one tracking issue and add that GH issue# here.

Reply:
Will create an issue.